### ISSUE TITLE
[libretro] [GBC] 007 - The World Is Not Enough - Character sprite sometimes invisible

### ISSUE THREAD

**Issue by Pixelnarium** [LIJI32/SameBoy#62]
I tested 007 with the libretro core and the character sprite is invisible depending on the position. It happens  directly in the first level when moving around.

This does **not** happen with the stand-alone SameBoy build from the website.

Tested the libretro core under Windows and Linux and stand-alone SameBoy under Windows.

**Comment by GrieverV** [comment]
Is this perhaps related to #61? It has been fixed in SameBoy master but the libretro fork is at cc95c89 which contains the problematic commit (d8e0683)

Might be worth testing d8e0683 standalone or cherry picking cb01259 to the libretro fork

**Comment by Pixelnarium** [comment]
I see. I thought libretro-sameboy and stand-alone sameboy are on the same version "10.1". I did not know that libretro-sameboy follows the master branch. I can confirm that this issue is a duplicate and already fixed in the master branch.

I could not build the libretro core right away. I got an undefined reference to "GB_debugger_clear_symbols" in gb.c while linking (on Linux). For testing purposes I just removed [that line](https://github.com/LIJI32/SameBoy/blob/master/Core/gb.c#L147).
